cookbook/debugging.rst

```diff
@@ -21,10 +21,9 @@ Disabling the Bootstrap File and Class Caching
 
 And to make the production environment as fast as possible, Symfony creates
 big PHP files in your cache containing the aggregation of PHP classes your
-project needs for every request. However, this behavior can confuse your IDE
-or your debugger. This recipe shows you how you can tweak this caching
-mechanism to make it friendlier when you need to debug code that involves
-Symfony classes.
+project needs for every request. However, this behavior can confuse your
```

Comment:
Here you explain that this can cause troubles ... but I'm missing the reason for it. What about rewording this:

```
However, this behavior can confuse your debugger.
```

to something like this:

```
However, this behavior can confuse your debugger, because the same class can be located in
two different places: the original class file and the big file which aggregates lots of classes.
```